packages/common/src/store/player/sagas.ts

```diff
@@ -9,9 +10,18 @@ import { PlaybackRate, PLAYBACK_RATE_LS_KEY } from './types'
  * Sets the playback rate from local storage when the app loads
  */
 function* setInitialPlaybackRate() {
+  const getFeatureEnabled = yield* getContext('getFeatureEnabled')
   const getLocalStorageItem = yield* getContext('getLocalStorageItem')
+  const isPodcastFeaturesEnabled =
+    ((yield* call(
```

Comment:
if you merge with main, a fix was added so you dont need to do any of this casting or ?? stuff, thank heavens